specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerConnections.json

```diff
@@ -0,0 +1,323 @@
+        "tags": [
+          "NetworkManagerConnections"
+        ],
+        "operationId": "NetworkManagerConnections_Delete",
+        "description": "Delete specified pending connection created by this subscription/management group.",
+        "responses": {
+          "200": {
+            "description": "Delete Succeed."
+          },
+          "204": {
+            "description": "Request successful. The resource does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Network Manager Connection": {
+            "$ref": "./examples/NetworkManagerConnectionDelete.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Network/networkManagerConnections": {
+      "get": {
+        "tags": [
+          "NetworkManagerConnections"
+        ],
+        "operationId": "NetworkManagerConnections_List",
+        "description": "List all network manager connections created by this subscription/management group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ListTopParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ListSkipTokenParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the network manager connection(s).",
+            "schema": {
+              "$ref": "#/definitions/NetworkManagerConnectionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List Network Manager Connection": {
+            "$ref": "./examples/NetworkManagerConnectionList.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Network/listPendingScopeConnections": {
+      "post": {
+        "tags": [
+          "NetworkManagerConnections"
+        ],
+        "operationId": "PendingScopeConnections_List",
+        "description": "List all pending scope connections created by a network manager.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ListTopParameter"
```

Comment:
ListPending is server-side pagination. It doesn't take query parameter top or skip. We need to send a post body with the optional skip token